cfme/configure/configuration/__init__.py

```diff
@@ -1885,6 +1886,20 @@ def unset_ntp_servers():
     return set_ntp_servers("", "", "")
 
 
+def configure_ntp_servers(*server_details):
```

Comment:
Personally, I feel this function is redundant and a little confusing. We already have a function called ```set_ntp_servers``` and all this function does is to serve up some default data. I would far prefer the functionality to be moved into the original function. I do however like the addition of the flash assertion and this should be migrated also. I'm not sure the unset_ntp_servers() is necessary. If the user does not specify 3 servers, then in one way I would say it is their fault. We could always pad out the ```*servers``` to always make it 3.